beater/test_approved_es_documents/TestPublishIntegrationTransactions.approved.json

```diff
@@ -335,7 +335,10 @@
             },
             "http": {
                 "request": {
-                    "method": "post"
+                    "method": "post",
+                    "socket": {
+                        "remote_address": "192.0.1"
```

Comment:
I am confused. Isn't this an illegal IP?

Reply:
Yes it is, but the `http.request.socket.remote_address` stores the original value. The sanitization is only applied to the `client.ip`, which is an ECS field, indexed, and used for further processing, such as the GeoIP pipeline.